### ISSUE TITLE
New Worker Pricing Example Mismatch

### ISSUE THREAD

**Issue by garretcharp** [cloudflare/cloudflare-docs#11015]
### Which Cloudflare product does this pertain to?

Workers

### Existing documentation URL(s)

- https://developers.cloudflare.com/workers/platform/pricing/#example-pricing-2

### What changes are you suggesting?

![image](https://github.com/cloudflare/cloudflare-docs/assets/32212214/d1a27259-8cda-4b47-8c77-6575305f8eab)

The example pricing for the new worker pricing says "7 milliseconds (ms) of CPU time per request" but then the table for calculating the total cost uses 200 ms CPU time per request.

I'm also not sure that the calculation used for CPU time there is even correct to begin with as it doesn't subtract the 30,000,000ms included and instead uses some calculation with the included requests. Unless I am misunderstanding the pricing with CPU time there.

**Comment by garretcharp** [comment]
![image](https://github.com/cloudflare/cloudflare-docs/assets/32212214/36c03308-383d-4f4b-aa44-02249bec1a12)

It does look like the actual costs are correct for the 100m/mo @ 7ms CPU so only the formula needs to be updated to:

```
(Total CPU Time - 30,000,000 ms included) / 1,000,000 * $0.02
```

**Comment by irvinebroque** [comment]
Addressed by https://github.com/cloudflare/cloudflare-docs/commit/b9dbf44427a077001c751b709925b5e42dd17dc1
